tidb-cloud/integrate-tidbcloud-with-netlify.md

```diff
@@ -32,32 +32,189 @@ For Dedicated Tier clusters, make sure that the traffic filter of the cluster al
+    (Use `node --trace-warnings ...` to show where the warning was created)
+    Deploy key added!
+
+    Creating Netlify GitHub Notification Hooks...
+    Netlify Notification Hooks configured!
+
+    Success! Netlify CI/CD Configured!
+
+    This site is now configured to automatically deploy from github branches & pull requests
+
+    Next steps:
+
+    git push       Push to your git repository to trigger new site builds
+    netlify open   Open the Netlify admin URL of your site
+    ```
 
-    - **Key** = DATABASE_URL
-    - **Values** = `mysql://<User>:<Password>@<Endpoint>:<Port>/<Database>?sslaccept=strict`
+3. Set environment variables. To connect to your TiDB Cloud cluster from your own space and the Netlify space, you need to set the `DATABASE_URL` as the connection string obtained from [Step 1](#step-1-get-the-example-project-and-the-connection-string).
 
-    You can get the information of `<User>`, `<Password>`, `<Endpoint>`, `<Port>`, and `<Database>` in the TiDB Cloud console.
+    ```shell
+    # set the environment variable for your own space
+    export DATABASE_URL='mysql://<User>:<Password>@<Endpoint>:<Port>/<Database>?sslaccept=strict'
 
-![Set an environment variable in Netlify](/media/tidb-cloud/integration-netlify-environment-variables.jpg)
+    # set the environment variable for the Netlify space
+    netlify env:set DATABASE_URL 'mysql://<User>:<Password>@<Endpoint>:<Port>/<Database>?sslaccept=strict'
+    ```
+
+    Check your environment variables.
+
+    ```shell
+    # check the environment variable for your own space
+    env | grep DATABASE_URL
+
+    # check the environment variable for the Netlify space
+    netlify env:list
+    ```
+
+4. Build the app locally and migrate the schema to your TiDB Cloud cluster.
+
+    > **Tips:**
+    >
+    > If you want to skip the local deployment and directly deploy the app to Netlify, just go to step 6.
+
+    ```shell
+    npm install .
+    npm run netlify-build
+    ```
+
+5. Run the application locally. You can start a local development server to preview your site.
+
+    ```shell
+    netlify dev
+    ```
+
+    Then, go to `http://localhost:3000/` in your browser to explore its UI.
+
+6. Deploy the app to Netlify. Once you are satisfied with the local preview, you can deploy your site to Netlify using the following command. `--trigger` means deployment without uploading local files. If you made any local changes, make sure that you have committed them to your GitHub repository.
+
+    ```shell
+    netlify deploy --prod --trigger
+    ```
 
-After re-deploying the site, you can use this new environment variable to connect to your TiDB Cloud cluster.
\ No newline at end of file
+    Go to your Netlify console to check the deployment state. After the deployment is done, the site for the app will have a public IP provided by Netlify so that everyone can access it.
```

Comment:
```suggestion
    Go to your Netlify console to check the deployment state. After the deployment is done, the site for the app will have a public IP address provided by Netlify so that everyone can access it.
```